src/sentry/static/sentry/app/views/settings/projectGeneralSettings.jsx

```diff
@@ -475,33 +476,37 @@ class ProjectGeneralSettings extends AsyncView {
             />
           )}
 
-          {organization.features.includes('datascrubbers-v2') ? (
-            <Panel>
-              <PanelHeader>{t('Data Privacy')}</PanelHeader>
-              <EmptyMessage
-                title="Data Privacy section now has its own tab 🎉"
-                description={
-                  <Link to={`/settings/${orgId}/projects/${projectId}/data-privacy/`}>
-                    {t('Go to Data Privacy')}
-                  </Link>
-                }
-              />
-            </Panel>
-          ) : (
-            <JsonForm
-              {...jsonFormProps}
-              title={t('Data Privacy')}
-              fields={[
-                fields.dataScrubber,
-                fields.dataScrubberDefaults,
-                fields.scrubIPAddresses,
-                fields.sensitiveFields,
-                fields.safeFields,
-                fields.storeCrashReports,
-                fields.relayPiiConfig,
-              ]}
-            />
-          )}
+          <Feature features={['datascrubbers-v2']}>
+            {({hasFeature}) =>
+              hasFeature ? (
+                <Panel>
+                  <PanelHeader>{t('Data Privacy')}</PanelHeader>
+                  <EmptyMessage
+                    title="Data Privacy section now has its own tab 🎉"
```

Comment:
Is this copy going to change? I'm not aware of us using emoji elsewhere in the product info boxes.